index.js

```diff
@@ -18,6 +17,7 @@ let metagen = dir => relativeFilenames(dir.path, dir.exclusions || [dir.output |
     .then(dir.filter || defaultFilter)
     .then(files => fs.writeFileAsync(dir.path + (dir.output || '__all.js'), dir.format(files, dir)))
 
+
```

Comment:
@agamemnus almost there 👍 Can you remove the extra line break you added in your merge conflict resolution?